### PR TITLE
Fix reconnect issue on Blade restart

### PR DESCRIPTION
**Pull request by lpradovera**

### CODE REVIEW

**Comment by lpradovera** [comment]
@bryanrite I checked this and ite behaves correctly against a Blade-side restart. GTG for now, we'll dig more if issues come up.

File: lib/signalwire/relay/consumer.rb [hunk 1]
```diff
@@ -70,13 +75,19 @@ def stop
   private
 
     def setup_receive_listeners
-      self.class.contexts.each do |cxt|
-        client.calling.receive context: cxt do |call|
+      self.class.contexts.each do |ctx|
```
Comment (bryanrite) [inline lib/signalwire/relay/consumer.rb:78]: 😄

**Review by bryanrite — CHANGES_REQUESTED** [review]
Looks good, just needs a CHANGELOG entry.

**Review by bryanrite — DISMISSED** [review]

**Comment by lpradovera** [comment]
@bryanrite re-approve please.

**Review by bryanrite — APPROVED** [review]

**Pull request merged**